pkgs/development/compilers/elm/default.nix

```diff
@@ -48,6 +63,11 @@ let
               jailbreak = true;
               # Tests are failing because of missing instances for Eq and Show type classes
               doCheck = false;
+
+              description = "Instrument Elm code as a preprocessing step. Meant to be using in conjunction with elm-coverage.";
```

Comment:
description should not contain dots at the end which probably means two sentences are to long.